message-window.lisp

```diff
@@ -150,47 +46,53 @@ function expects to be wrapped in a with-state for win."
           (unless (frame-window frame)
             (draw-frame-outline group frame t t)))))))
 
+(defclass frame-indicator-window (stumpui:text-window
+                                  stumpui:timed-window)
+  ()
+  (:default-initargs :event-mask '(:exposure)
+                     :padding 0))
+
 (defun show-frame-indicator (group &optional force)
   (show-frame-outline group)
-  ;; FIXME: Arg, these tests are already done in show-frame-outline
-  (when (find group (mapcar 'screen-current-group *screen-list*))
+  ;; FIXME: Arg, this test is already done in show-frame-outline
```

Comment:
We can, by making `show-frame-outline` return whether or not it actually did anything. I updated the comment because "these tests" was confusing in the presence of other tests that aren't done in `show-frame-outline`.